api/models/Reading.js

```diff
@@ -6,7 +6,23 @@ const readingSchema = new mongoose.Schema({
   value: {
     required: true,
     type: String
+  },
+  tankId: {
+    required: true,
```

Comment:
@ArpitKotecha Can you please review this. I see similar PR at other places too? Can you please verify.

Reply:
@vinitshahdeo Probably while creating/modifying routes in other PR's they have modified the schema accordingly.

Reply:
> @ArpitKotecha Can you please review this. I see similar PR at other places too? Can you please verify.

@vinitshahdeo Maybe we can merge this and request in other PR to include this schema instead of modifying by themselve.